src/native/libs/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -144,6 +144,13 @@ void SSL_CTX_set_alpn_select_cb(SSL_CTX* ctx,
 void SSL_get0_alpn_selected(const SSL* ssl, const unsigned char** protocol, unsigned int* len);
 #endif
 
+#if !HAVE_OPENSSL_ALPN
```

Comment:
This should not be needed.

Reply:
This should have been `HAVE_OPENSSL_PHA`.

Anyway, how else should I make the code compilable on systems with OpenSSL older than 1.1.1?